<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="interfaceWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>721</width>
     <height>551</height>
    </rect>
   </property>
   <widget class="QWidget" name="login">
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enrollButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>350</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>491</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>欢迎使用网络聊天工具</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="userName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="userPwd">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userNameEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="userPwdEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="enroll">
    <widget class="QLabel" name="enrollUserName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="enrollUserPwd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="enrollUserNameEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="enrollUserPwdEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="enrollSummitButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>400</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="user"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
